Documentation/ceph-upgrade.md

```diff
@@ -500,7 +396,7 @@ The majority of the upgrade will be handled by the Rook operator. Begin the upgr
 Ceph image field in the cluster CRD (`spec:cephVersion:image`).
 
 ```console
-NEW_CEPH_IMAGE='ceph/ceph:v14.2.2-20190830'
+NEW_CEPH_IMAGE='ceph/ceph:v14.2.5-20191210'
```

Comment:
shall we use `ceph/ceph:v14.2.5` to be consistent with the sample cluster.yaml in 1.2?

Reply:
We shouldn't use `v14.2.5` so that users don't end up having mixed container versions in production.